requirements.txt

```diff
@@ -5,4 +5,6 @@ notebook
 s3fs
 tornado >= 5.1.1
 aioboto3
-aiobotocore>=0.10.0
+aiobotocore>=0.10.2
+boto3==1.9.91
```

Comment:
Is there a reason we're tightly pinning here? This will cause build issues for many systems that control boto version across many projects.

Reply:
There are some dependency issues among aioboto3, aiobotocore, boto3, and botocore. I can't seem to get the dependencies to resolve (needed for autodoc) without pinning. That said, I agree with you this is less than ideal, and I'm all ears if you have a workaround.

Reply:
Ahh ok. So I talked with our image build team and they're super frustrated by the current boto release process. To the point that they ignore pins on boto related projects to even get anything to build in tandem.

There's a few ways we can approach this.
1) Only pin for the autodoc build tool. Make a more specific requirements-doc.txt which pins more specifically.
2) Make the boto imports lazy (inside classes) so we don't need boto for doc builds at all.

I think 1) is simpler for now and keeps the tool from aggressive pinning

Reply:
I will try 1 in the next few days. Agree with the image build team.